docs/compression.md

```diff
@@ -0,0 +1,97 @@
+# Imagery Compression
+
+
+## Background
+
+Toitū Te Whenua (LINZ) holds multiple imagery types data for [most of New Zealand](https://www.linz.govt.nz/products-services/data/types-linz-data/aerial-imagery).
```

Comment:
> Page cannot be found (Error 404)

Reply:
are you looking at this inside of LINZ? as the LINZ website doesn't seem to work well inside of LINZ?

Reply:
Agh, this again!

Reply:
Ill assume that means yes you are inside linz, as these links work fine for me.